### PR TITLE
Update Project

### PR DESCRIPTION
**Pull request by LauLaman**
- Update Dependencies
- Update PHPUnit tests

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: php
 
 php:
-  - '7.1'
+  - '7.3'
```
Comment (thijsBreker) [inline .travis.yml:4]: I take it going to php 7.4 is a step too far? (Also: Hey Laurens!)

Comment (LauLaman) [reply .travis.yml:4]: Hi Thijs,
You are right! we should test also in PHP 7.4

**Comment by LauLaman** [comment]
@thijsBreker Can you make sure Travis runs and scrutinizer is updated to the correct PHP version?

**Comment by ferrastas** [comment]
Hey @LauLaman we just fixed the integration for travis, could you please rebase your branch?

**Comment by LauLaman** [comment]
@ferrastas done!

Also, i noticed that [packagist](https://packagist.org/packages/werkspot/enum) is not auto updated 💁‍♂️

**Review by ferrastas — APPROVED** [review]

**Comment by ferrastas** [comment]
Thanks @LauLaman we will check the integration to packagist.

**Comment by LauLaman** [comment]
@ferrastas/@thijsBreker Can we also have a new release for this change? like `3.0`?

**Pull request merged**